File: FCUBEAPI/FreightMasters/Models/AdditionalCostRecMasterList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FreightMasters.Models
{
    public class AdditionalCostRecMasterList
    {
        public List<AdditionalCostRecMasterModel> AdditionalList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/additionalcostrecmasterlist.ts
import { Paginationmodel } from "./paginationmodel";
import { AdditionalcostrecmasterModel } from "./additionalcostrecmastermodel";

export class Additionalcostrecmasterlistmodel {
    additionalList: AdditionalcostrecmasterModel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}